docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -27,6 +27,8 @@ Then, install the Amplify UI React package:
 ### Basic Demo
 
 <Alert variation="info" role="none">
+Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
```

Comment:
Talked with @reesscot offline, we removed the redirect from to troubleshooting here, because it gives a notion that this is broken. So I moved the instruction to both places, which is a fair bit of duplication, but this makes sense for customers visiting this page.